addon/utils/is-descriptor.js

```diff
@@ -2,7 +2,14 @@ import Ember from 'ember';
 let { typeOf, Descriptor } = Ember;
 
 // credit: https://github.com/cibernox/ember-cpm/blob/master/addon/utils.js#L17-L20
-function isDescriptor(prop) {
+function isDescriptor(propertyName) {
```

Comment:
ha, no. I think this is the current insanity. Post 2.0 we will likely stuff the CP desc back into meta ...